.gitignore

```diff
@@ -3,5 +3,7 @@
 .packages
 .pub/
 build/
+.idea
+pubspec.lock
```

Comment:
We explicitly DO checkin the pubspec.lock file – to track which dependencies we use